docs/main/docs/intro.md

```diff
@@ -5,19 +5,49 @@ id: intro
+1. **Multichain Smart Contract Executions:**
+   - t3rn enables smart contracts to perform transactions across multiple blockchain networks, ensuring fail-safe operations and expanded functionality.
+1. **Support for Multiple Languages:**
+   - Developers can write smart contracts in Solidity, !ink, or any language that compiles to WASM. t3rn supports both WASM and EVM, offering versatility in development.
+1. **Open-Source Registry:**
+   - Smart contracts on t3rn are stored in an open-source registry, making them accessible for reuse and development by the broader community.
+1. **Remuneration for Creators:**
+   - Contract creators are [rewarded](about_t3rn/token-economics/tokenomics-inflation#gas-fees) each time their code is executed, receiving a share of the gas fee. This incentivizes open-source development and fosters a new paradigm of collaborative creation.
+
+## Key Network Participants
+
+To facilitate its multichain transactions, t3rn relies on essential network participants:
+
+1. **Executors:**
+   - Executors play a crucial role in processing cross-chain transactions. They assess, bid on, and execute orders created by users, ensuring efficient transaction completion.
+1. **Attesters:**
+   - Attesters are responsible for verifying the correctness and completion of cross-chain transactions. They submit attestations that are crucial for the validation and finalization of transactions.
+1. **Collator:**
+   - Callators in the t3rn ecosystem are vital for facilitating cross-chain communication and transactions. They relay transaction information between chains, ensuring accurate and efficient execution of cross-chain calls.
+
+## Getting Started with t3rn:
+
+1. **Collator Node Operation:**
+   - To participate in the t3rn network, you can set up and run a collator node. This involves installing the software, configuring settings, and syncing with the desired network (e.g., t2rn, t0rn, t1rn, or t3rn).
+1. **Smart Contract Development:**
+   - Develop smart contracts using Solidity or any WASM-compatible language. Deploy your contracts to t3rn’s open-source registry to make them available for multichain execution and collaboration. Learn more about the [t3rn Smart Contract Hub](https://www.notion.so/Docs-Smart-Contract-Hub-b0095284acf14a7faa6a283929b2797d?pvs=21).
+1. **Participate as an Executor or Attester:**
+   - Join the network as an Executor to process transactions or as an Attester to validate them. Engage in the ecosystem by bidding on orders, executing transactions, and submitting attestations.
+1. **Explore t3rn's Capabilities:**
+   - Dive into the world of cross-chain swaps, understand the functionalities of the Remote Order, EscrowGMP, and Attestations Verifier contracts, and explore how t3rn solves blockchain connections.
+
+## Popular 🔥
+
+- **Brain explanation** coming soon.
```

Comment:
You can update this and the "Smart Contract Hub" after merging the other two PRs.